fighthealthinsurance/common_view_logic.py

```
@@ -401,9 +401,9 @@ def _assemble_appeal_pdf(
         if pubmed_ids_parsed is not None and len(pubmed_ids_parsed) > 0:
             logger.debug(f"Processing PubMed articles: {pubmed_ids_parsed}")
             pmt = PubMedTools()
-            pubmed_docs: list[PubMedArticleSummarized] = pmt.get_articles(
-                pubmed_ids_parsed
-            )
+            pubmed_docs: list[PubMedArticleSummarized] = async_to_sync(
```

Ruff (0.8.2)

`PubMedArticleSummarized` may be undefined, or defined from star imports

(F405)